.gitignore

```diff
@@ -43,3 +43,4 @@ testem.log
 /.git
 /.firebase
 /.bazelrc.user
+/.vs
```

Comment:
What is this for? I wouldn't change anything outside the scope of the task

Reply:
This was get added by VS while committing code. I am going to deleting it now.